packages/shared/create-local-vue.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import Vue from 'vue'
-import cloneDeep from 'lodash/cloneDeep'
+import { cloneDeep, merge } from 'lodash/cloneDeep'
```

Comment:
Hm, I'm not sure you can import lodash packages from `lodash/cloneDeep` like this, but rather

```js
import { cloneDeep, merge } from 'lodash'
```

Reply:
I think this will not tree shake well - it will include *all* of `lodash` in the final build (maybe? lodash pre-dates tree shaking).

I think what we should be doing is this:

```
import cloneDeep from 'lodash/cloneDeep'
import merge from 'lodash/merge'
```

The best way to verify this is just doing all the different imports and seeing how the final bundle changes

Reply:
sure! I think it depends on bundlers, too. Also, VTU is used in dev time (not production), so build size isn't _that_ important, but if we're faced with to similar alternatives, one being lighter… let's go with it :)

Reply:
good point @afontcu I didn't pay attention to that